constant.go

```diff
@@ -0,0 +1,9 @@
+package gorocksdb
+
+const (
+	KB int = 1024
+	MB int = KB * KB
```

Comment:
Better to use `KiB`, `MiB` etc to avoid confusion with the metric variants.  Also these are better as untyped constants. So the result would be this:

```go
const (
	KiB = 1024
	MiB = KiB * KiB
	GiB = KiB * MiB
	TiB = KiB * GiB
	PiB = KiB * TiB
)
```